### ISSUE TITLE
False-positive disk size error in super/Pie/arm64

### ISSUE THREAD

**Issue by yurikoles** [opengapps/opengapps#686]
**Only issues related to the code of The Open GApps Project scripts themselves**

If you have found an issue with a pre-built OpenGApps.org package please use the XDA-Developers [forum thread](http://forum.xda-developers.com/android/software/Open-GApps-t3098071)

**Checklist**

- [x] The issue is not answered by reading the [FAQ](https://github.com/opengapps/opengapps/wiki/FAQ)
- [x] The issue is part of The Open GApps Project scope *(and not of the pre-built OpenGApps.org packages scope, for support on those see [XDA Forum](http://forum.xda-developers.com/android/software/Open-GApps-t3098071))*

*If Open GApps installation fails:*
- [x] Is the issue reproducible using the latest version of your recovery?
- [x] Is the issue reproducible with (the latest) [TWRP](https://twrp.me) recovery? *(other recoveries are only best-effort)*

*If receiving force closures after Open GApps installation:*
- [x] The issue is **NOT** an error within one of the bundled Google applications itself *(verify by e.g. installing the application from the Play Store instead)*
- [x] The issue is reproducible on clean (=wiped ``/data/`` partition) installation
- [x] The affected ROM includes the necessary patches and functionality as noted for [Android versions](https://github.com/opengapps/opengapps/wiki#notes-for-android-versions) and [Platforms](https://github.com/opengapps/opengapps/wiki#notes-for-platforms)
- [x] No non-AOSP hacks like privacy guards' permission restrictions etcetera are applied

**Issue description:**

Installer reports insufficient space. But I'm able to install after removing size calculation from script and still have about 80MB left after install.

**Reproduction Steps:**

1. Try to install super/Pie/arm64 package

Size calculation is correct.

**Describe expected behavior here**

Size calculation reports not enough space.

**Describe observed behavior here**

I see false-positive disk calculation error.

[Debug log of the Open GApps installation](https://github.com/opengapps/opengapps/files/2539363/open_gapps_debug_logs.tar.gz)

**Comment by NatoBoram** [comment]
I can confirm, there's a false-positive disk calculation error when flashing OpenGApps on Android 9. It's not exclusive to the Super Package.

I tried to flash OpenGApps Stock over [Stock Android](https://developers.google.com/android/images#sailfish) and I got error `70`. I then tried smaller and smaller packages, and finally, I tried Pico. Still error `70`.

```yaml
Board : sailfish
TWRP : 3.2.3-1-sailfish
OpenGApps : Every single one. No exception.
```
